core/server/themes/loader.js

```diff
@@ -1,40 +1,50 @@
 var debug = require('debug')('ghost:themes:loader'),
     config = require('../config'),
     events = require('../events'),
+    list = require('./list'),
     read = require('./read'),
-    settingsApi = require('../api/settings'),
     settingsCache = require('../settings/cache'),
-    updateConfigAndCache,
-    loadThemes,
+    updateThemeList,
+    loadAllThemes,
+    loadOneTheme,
     initThemes;
 
-updateConfigAndCache = function updateConfigAndCache(themes) {
+updateThemeList = function updateThemeList(themes) {
     debug('loading themes', Object.keys(themes));
-    config.set('paths:availableThemes', themes);
-    settingsApi.updateSettingsCache();
+    list.init(themes);
```

Comment:
For better occurrences lookup, i would suggest we keep one naming pattern.
Sometimes we use `themeList.get`, sometimes `themeUtils.list..`, sometimes `list.init`.